src/data/ores/bronzeOre.js

```diff
@@ -0,0 +1,16 @@
+import { itemType } from '../../enums'
+
+const { freeze } = Object
+
+/**
+ * @property farmhand.module:items.bronzeOre
+ * @type {farmhand.item}
+ */
+export const bronzeOre = freeze({
+  description: 'A piece of bronze ore you dug up in the field.',
```

Comment:
I like the `A piece of bronze ore` part of this, but I'm not sure about the `you dug up in the field` part. I could see ores being being obtained a number of ways (such as an achievement reward), so that part of the description would be inaccurate in that case. What do you think about removing of rewording it?

(Same for the other new ores.)

Reply:
works for me!